pandas/tests/indexes/common.py

```diff
@@ -10,7 +10,7 @@
 from pandas.core.dtypes.dtypes import CategoricalDtype
 
 import pandas as pd
-from pandas import (
+from pandas import (  # noqa
```

Comment:
why noqa?

Reply:
`Int64` is not called directly, but implicitly in a `eval` call in `test_repr_roundtrip`.